.github/workflows/greetings.yml

```diff
@@ -0,0 +1,14 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Hola! @${{ github.actor }} 🥳, You just created an Issue!🌟 Thanks for making the Project Better and empowering Learners'
```

Comment:
Lets do: 
"Hola! @${{ github.actor }} 🥳, You've just created an Issue!🌟 Thanks for making the Project Better"